confluent_kafka/src/Producer.c

```diff
@@ -251,7 +293,8 @@ static PyObject *Producer_produce (Handle *self, PyObject *args,
 	int value_len = 0, key_len = 0;
 	int partition = RD_KAFKA_PARTITION_UA;
 	PyObject *dr_cb = NULL, *dr_cb2 = NULL, *partitioner_cb = NULL;
-	rd_kafka_topic_t *rkt;
+        long long timestamp = 0;
```

Comment:
Also, is setting this to 0 by default the correct behavior? If we are always going to force use of CreateTime, -1 seems more consistent with KIP-32 (since that's what's filled in for earlier produce request versions). Alternatively, should we be including support for LogAppendTime vs CreateTime?